### ISSUE TITLE
[Feature Request] Email report metrics

### ISSUE THREAD

**Issue by troelsfeodor** [mautic/mautic#990]
The data you currently are shown for a sent email is very lacking. It would be good to at least show industry standard metrics like
- delivery rate
- open rate
- click rate
- unsubscribes

**Comment by that0n3guy** [comment]
I agree,  We need:

- Opens
- Clicks	 
- Unsubscribe	 
- Hard Bounces	 
- Spam 
- Soft Bounces
- Unique Opens 
- Unique Clicks

**Comment by npracht** [comment]
We'd also need in <b>one click</b> to see the list of the leads with those filters.
Who clicked, who opened, who unsubscribed...

**Comment by that0n3guy** [comment]
Another simple example:
![image](https://cloud.githubusercontent.com/assets/113870/10639366/75481eb8-77d5-11e5-9f7e-ea9881fdc9c9.png)

**Comment by SKiD12** [comment]
It would be nice to have these metrics in each e-mail stats and also in Reports

**Comment by ramezrafla** [comment]
Agreed, here is the report screenshot of the best tools out there, sendy:

![sendy-report](https://cloud.githubusercontent.com/assets/1903864/12697094/9cfa7a56-c748-11e5-879c-4fecb4f30c84.jpg)

All the info is already available in the DB, so it's a matter of loading and presenting.

Reference: https://sendy.co/images/sendy-report.jpg?6

**Comment by healthstatus** [comment]
+1

**Comment by ptcc** [comment]
:+1:

**Comment by dailybits** [comment]
+1

**Comment by Yame-** [comment]
+1

**Comment by theosyslack** [comment]
_+1_

**Comment by dbrown546** [comment]
+1

**Comment by escopecz** [comment]
This PR for Mautic 2.0 will improve the email stats a bit: https://github.com/mautic/mautic/pull/1713

**Comment by MxyzptlkFishStix** [comment]
Useless metrics.

**Comment by escopecz** [comment]
@MxyzptlkFishStix Could you please break down what do you mean? What comment are you referring to? You managed to write an useless comment.

**Comment by ramezrafla** [comment]
Absolutely! @MxyzptlkFishStix comment was not only worthless, it actually wasted the time of many. 

On another note, we are excited at the report metrics improvement. It may even open the window to the community creating their own reports, which we can contribute back to core. 

When is 2.0 planned for? Can't find a roadmap for Mautic.

**Comment by MxyzptlkFishStix** [comment]
@ramezrafla If two short words took more time to process than an eye blink, I don't know what to say.

=====

@escopecz The reason why I stated that is because anyone that has an even rudimentary understanding of emailing knows that opens & clicks are poor metrics to measure the success of any email campaign.  Well, unless you're a solo ad seller or offer a SMTP SAAS with pretty graphics to customers.

Just looking at opens alone as a metric to measure:
- the web bug is blocked by the client
- the web bug is cached by the ISP due to having no-cache headers
- multiple opens not taken in account by the tracking platform
- incorrectly counting an open when the client software pre-loads the email that the person never reads
- you can't tell by an open if they read the first line, the first paragraph or the email in it's entirety

Those things alone can greatly skew your "data".  Reliance on poor metrics can be the difference between a failed campaign and a 5 or 6 figure one.  It'll get even worse as more and more people are viewing email on their mobile devices which a great many apps block image loading by default. The focus should be on what is called CTC (Click-To-Conversion).  You're going to have to figure out a way to tie your email with your website analytics.

**Comment by escopecz** [comment]
@MxyzptlkFishStix Thanks for a slightly longer and definitely more valuable comment. I agree that the email open metric is not 100% accurate it may not even be 60% accurate, but it is still better than not tracking it at all.

On the other hand the email click metric is quite accurate. I'm sure you know how it works. Just for others; Mautic will replace the links in the email with unique links to itself before the email is sent. When a contact clicks any link in the email, she is redirected to Mautic itself. Mautic will track the link click event and redirect contact to the original URL.

**Comment by tristanbailey** [comment]
+1

**Comment by MxyzptlkFishStix** [comment]
@escopecz 

Is Mautic using the formula (Confirmed Open Rate) below to determine open rates?

(**Opens** + **Clicks w/o Opens**) / (**Total Emails Sent** - **Bounces**)

**Comment by escopecz** [comment]
@that0n3guy did not know that. Thanks for info.

@MxyzptlkFishStix No, Mautic does not count the open rate. But it's a great suggestion.

**Comment by thelonious89** [comment]
I see we're coming up to 1 year since this request was first made. :)

Mautic is fantastic. It is easy, and fluid, and graceful. I like it. But it does need more finer tuned controls over reporting. I understand and appreciate the feedback from @MxyzptlkFishStix and it's a struggle to explain 1x1 tracking gifs to unsophisticated clients, but in general it's much, much, much better to have that info on opens, clicks, etc. than not to have it.

CTC is a great method to gauge a campaign's progress, but it's not useful in all campaigns or for all clients. Lots of times, tracking conversions is very difficult or expensive to track if the conversion is supposed to happen offline (at a retail store, for example). And other times, a campaign doesn't have anything to really convert -- it is meant as an announcement, a newsletter, or something else where actually knowing the deliverability (hard and soft bounces) is more important than knowing if someone opened or clicked.

**Comment by namratasaun** [comment]
I can see sent, unsubscribed, opened, bounced as metrics. Is there any way I can get spam reports asa. metric

**Comment by exsesx** [comment]
Any updates?
